onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperatorResize.cpp

```diff
@@ -307,25 +302,35 @@ class DmlOperatorResize : public DmlOperator, public ResizeHelper
         std::vector<DML_TENSOR_DESC> inputDescs = GetDmlInputDescs();
         std::vector<DML_TENSOR_DESC> outputDescs = GetDmlOutputDescs();
 
-#if DML_TARGET_VERSION >= 0x6400
-        DML_RESAMPLE3_OPERATOR_DESC operatorDesc = {};
-        operatorDesc.Antialiased = static_cast<BOOL>(antialiased);
-#else
-        DML_RESAMPLE2_OPERATOR_DESC operatorDesc = {};
-#endif
-        operatorDesc.InputTensor = inputDescs.data();
-        operatorDesc.OutputTensor = outputDescs.data();
-        operatorDesc.InterpolationMode = interpolationMode;
-        operatorDesc.RoundingDirection = roundingDirection;
-        operatorDesc.Scales = paddedScales.data();
-        operatorDesc.DimensionCount = gsl::narrow_cast<uint32_t>(paddedScales.size());
-        operatorDesc.InputPixelOffsets = inputPixelOffsets.data();
-        operatorDesc.OutputPixelOffsets = outputPixelOffsets.data();
-#if DML_TARGET_VERSION >= 0x6400
-        DML_OPERATOR_DESC opDesc = { DML_OPERATOR_RESAMPLE3, &operatorDesc };
-#else
-        DML_OPERATOR_DESC opDesc = { DML_OPERATOR_RESAMPLE2, &operatorDesc };
-#endif
+        DML_OPERATOR_DESC opDesc = {};
+        if (opsetVersion >= 18) {
+          // Restrict this change to Resize18 and Resize19
+          const int antialiased = kernelCreationContext.GetOptionalAttribute<int>(AttrName::Antialiased, 0);
+          DML_RESAMPLE3_OPERATOR_DESC operatorDesc = {};
+          operatorDesc.Antialiased = static_cast<BOOL>(antialiased);
+          operatorDesc.InputTensor = inputDescs.data();
+          operatorDesc.OutputTensor = outputDescs.data();
+          operatorDesc.InterpolationMode = interpolationMode;
+          operatorDesc.RoundingDirection = roundingDirection;
+          operatorDesc.Scales = paddedScales.data();
+          operatorDesc.DimensionCount = gsl::narrow_cast<uint32_t>(paddedScales.size());
+          operatorDesc.InputPixelOffsets = inputPixelOffsets.data();
+          operatorDesc.OutputPixelOffsets = outputPixelOffsets.data();
+          opDesc = { DML_OPERATOR_RESAMPLE3, &operatorDesc };
+        }
+        else {
+          DML_RESAMPLE2_OPERATOR_DESC operatorDesc = {};
+          operatorDesc.InputTensor = inputDescs.data();
+          operatorDesc.OutputTensor = outputDescs.data();
+          operatorDesc.InterpolationMode = interpolationMode;
+          operatorDesc.RoundingDirection = roundingDirection;
+          operatorDesc.Scales = paddedScales.data();
+          operatorDesc.DimensionCount = gsl::narrow_cast<uint32_t>(paddedScales.size());
+          operatorDesc.InputPixelOffsets = inputPixelOffsets.data();
+          operatorDesc.OutputPixelOffsets = outputPixelOffsets.data();
+          opDesc = { DML_OPERATOR_RESAMPLE2, &operatorDesc };
+        }
```

Comment:
Observe intra-file consistency 👀 - see a few lines above uses 4-column indents, and so 4-column indents here too.

```suggestion
        DML_OPERATOR_DESC opDesc = {};
        if (opsetVersion >= 18)
        {
            // Restrict this change to Resize18 and Resize19.
            const int antialiased = kernelCreationContext.GetOptionalAttribute<int>(AttrName::Antialiased, 0);
            DML_RESAMPLE3_OPERATOR_DESC operatorDesc = {};
            operatorDesc.Antialiased = static_cast<BOOL>(antialiased);
            operatorDesc.InputTensor = inputDescs.data();
            operatorDesc.OutputTensor = outputDescs.data();
            operatorDesc.InterpolationMode = interpolationMode;
            operatorDesc.RoundingDirection = roundingDirection;
            operatorDesc.Scales = paddedScales.data();
            operatorDesc.DimensionCount = gsl::narrow_cast<uint32_t>(paddedScales.size());
            operatorDesc.InputPixelOffsets = inputPixelOffsets.data();
            operatorDesc.OutputPixelOffsets = outputPixelOffsets.data();
            opDesc = { DML_OPERATOR_RESAMPLE3, &operatorDesc };
        }
        else
        {
            DML_RESAMPLE2_OPERATOR_DESC operatorDesc = {};
            operatorDesc.InputTensor = inputDescs.data();
            operatorDesc.OutputTensor = outputDescs.data();
            operatorDesc.InterpolationMode = interpolationMode;
            operatorDesc.RoundingDirection = roundingDirection;
            operatorDesc.Scales = paddedScales.data();
            operatorDesc.DimensionCount = gsl::narrow_cast<uint32_t>(paddedScales.size());
            operatorDesc.InputPixelOffsets = inputPixelOffsets.data();
            operatorDesc.OutputPixelOffsets = outputPixelOffsets.data();
            opDesc = { DML_OPERATOR_RESAMPLE2, &operatorDesc };
        }
```